tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -112,7 +112,8 @@ def is_box_outputs(config, box_outputs):
         self.outputs_verified = False
 
     def select_output_blob(self, outputs):
-        self.raw_masks_out = self.check_output_name(self.raw_masks_out, outputs)
+        if self.raw_masks_out:
+            self.raw_masks_out = self.check_output_name(self.raw_masks_out, outputs)
         if hasattr(self, 'detection_out'):
             self.detection_out = self.check_output_name(self.raw_masks_out, outputs)
```

Comment:
One more bugfix for this adapter
```suggestion
            self.detection_out = self.check_output_name(self.raw_masks_out, outputs)
            return
```